test/pipeline-steps.test.ts

```diff
@@ -73,6 +73,15 @@ describe("extractFilesetFromPath", () => {
     ).toBe(false);
   });
 
+  test("should return manifest file when given manifest file path directly", () => {
+    const manifestPath =
+      "test/fixtures/manifest-only-package/Manifest.ocf.json";
+    const result = extractFilesetFromPath(manifestPath);
+    expect(result.length).toBe(1);
+    expect(result[0].path).toBe(manifestPath);
+    expect(result[0].readAsText()).toBe(fs.readFileSync(manifestPath, "utf8"));
+  });
```

Comment:
Good that we have a test, but this test doesn't exercise the `referencedFileObjs = referencedFiles.map` bit of code... I'd suggest that should be another test!